src/invariant/CacheIsConsistentWithDatabase.cpp

```diff
@@ -16,11 +17,13 @@ std::shared_ptr<Invariant>
 CacheIsConsistentWithDatabase::registerInvariant(Application& app)
 {
     return app.getInvariantManager()
-        .registerInvariant<CacheIsConsistentWithDatabase>(app.getDatabase());
+        .registerInvariant<CacheIsConsistentWithDatabase>(
+            app.getDatabase(), app.getLedgerManager());
 }
 
-CacheIsConsistentWithDatabase::CacheIsConsistentWithDatabase(Database& db)
-    : Invariant(false), mDb{db}
+CacheIsConsistentWithDatabase::CacheIsConsistentWithDatabase(Database& db,
+                                                             LedgerManager& lm)
+    : Invariant(false), mDb{db}, mLedgerManager(lm)
```

Comment:
Note: false here is cryptic, should be replaced with enum when that makes sense.